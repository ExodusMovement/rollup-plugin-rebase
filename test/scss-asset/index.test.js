import { bundle, clean, list, read } from "../util"

const root = __dirname

test("SCSS Asset", async () => {
  await bundle(root, "index.js", "output/index.js")

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/hdeAIjQB.scss")).toMatchSnapshot()

  await clean(root, "output")
})
