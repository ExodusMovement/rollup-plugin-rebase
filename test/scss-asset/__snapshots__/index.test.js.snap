// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SCSS Asset 1`] = `
"bXipeBrs.png
bsdudxaF.md
gToHHcDx.scss
gToHHcDx.scss.map
index.js"
`;

exports[`SCSS Asset 2`] = `
"import content from './bsdudxaF.md';
import scss from './gToHHcDx.scss';

function index() {
  return content + scss
}

export default index;
"
`;

exports[`SCSS Asset 3`] = `
"body {
  font-size: 20px;
  background: url(./bXipeBrs.png) top left repeat;
}

/*# sourceMappingURL=gToHHcDx.scss.map */"
`;
